stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/options/message/internal/MessageOptionsDialogFragment.kt

```diff
@@ -214,9 +214,21 @@ internal class MessageOptionsDialogFragment : FullScreenDialogFragment() {
                     isMessagePinned = message.pinned
                 )
             }
+
+            updateLayoutParams<ViewGroup.MarginLayoutParams> {
+                if (messageItem.isMine) {
+                    marginEnd = style.itemStyle.messageEndMargin + optionsOffset()
```

Comment:
If I add end margin on my items and set a predicate that displays avatars on my own messages, it'll now add a margin onto the message options as well, but not display my avatar there:

![image](https://user-images.githubusercontent.com/12054216/130482048-38cd9176-7201-4bf7-98de-bd36306db944.png)

Reply:
We add `Position.Bottom` to the `MessageItem` when we create it for this dialog... that's why the avatar doesn't appear in this case. 

Since the change the `MessageItem` of the dialog, I don't think we can include  messages in this dialog it in the Predicate. The way I see to deal with it is to create a separate configuration to control the avatar show in this specific case...